tests/cylc-message/00-ssh/suite.rc

```diff
@@ -9,9 +9,8 @@
 [runtime]
     [[t0]]
         script = """
-# Ensure that contact 2 is installed, but passphrase isn't.
+# Ensure that contact 2 is installed.
 test -f "${CYLC_SUITE_RUN_DIR}/.service/contact2"
```

Comment:
Does this line need to be removed?

Reply:
I have removed this test - task communication method = ssh is not current;y working and this will need fixing for #3327.